lib/tasks/sac_imports.rake

```diff
@@ -16,13 +16,15 @@ namespace :sac_imports do
     additional_memberships: "tmp/xlsx/zusatzmitgliedschaften.xlsx"
   }
 
-  desc "Import all people from a navision export xlsx" \
-         " (options: FILE=#{default_files[:people]} REIMPORT_ALL=true)"
-  task "1_people": [:environment] do
-    SacImports::PeopleImporter.new(
-      read_file(:people),
-      skip_existing: skip_existing?
-    ).import!
+  task override_truemail_configuration: [:environment] do
```

Comment:
hätte ich jetzt eher einfach in eine eigene methode hier ausgelagert anstatt einen eigenen task dafür